salt/auth/django.py

```diff
@@ -95,21 +111,24 @@ def django_auth_setup():
         django_model_name = django_model_fullname.split('.')[-1]
         django_module_name = '.'.join(django_model_fullname.split('.')[0:-1])
 
-        __import__(django_module_name, globals(), locals(), 'SaltExternalAuthModel')
+        django_auth_module = __import__(django_module_name, globals(), locals(), 'SaltExternalAuthModel')
         DJANGO_AUTH_CLASS_str = 'django_auth_module.{0}'.format(django_model_name)
         DJANGO_AUTH_CLASS = eval(DJANGO_AUTH_CLASS_str)  # pylint: disable=W0123
 
-    if django.VERSION >= (1, 7):
-        django.setup()
-
 
 def auth(username, password):
     '''
     Simple Django auth
     '''
-    import django.contrib.auth  # pylint: disable=import-error
+    sys.path.append(__opts__['django_auth_path'])
```

Comment:
Django is in PYTHONPATH. 
`django_auth_path` here means the django auth project path.

If I don't add this line here but set `PYTHONPATH` in `/etc/profile`, salt-master can't import the project's settings.

The document https://docs.saltstack.com/en/latest/ref/auth/all/salt.auth.django.html is not so detailed. I created a project https://github.com/hujunya/salt_django_auth and explain how to configure the django auth.

Does I use the wrong way or do you have the detailed usage about the django auth?